### PR TITLE
[Fix] CSS fixes

### LINKED ISSUE #33: Test app with modern browsers
Objective: Find issues with CSS and UX before sending app to client

Targets:

- [x] Firefox
- [x] Opera
- [x] Edge
- [x] Safari

### LINKED ISSUE #49: Do not move content if scrollbar is not preset
- Add margin when no scrollbar visible (reserve space)

### LINKED ISSUE #50: Add hover helpers
image
image

### LINKED ISSUE #54: Import to react css changes for scrollbar and tooltip
- [x] Build css classes and html
- [x] Import project into react files

### PR DESCRIPTION
**Pull request by SferaDev**
### :pushpin: References

-   **Issue:** Closes #33 
-   **Issue:** Closes #49
-   **Issue:** Closes #50 
-   **Issue:** Closes #54

### :memo: Implementation

- [FloatingButton] Fix centered on safari
- [CardProgress] Fix style on firefox
- [ModalContent] Fix scrollbar on firefox
- [Tooltip] Add floating tooltip for help

### :fire: How to test it? (If there is any special consideration or the reviewer does not know how to test it)

- Run the app
- Open the app on firefox and visit the OverviewPage and review the progress bar styling
- Open the app on firefox and visit a Tutorial step that has a scrollbar and review the scroll bar styling
- Open the app and visit a Tutorial step and hover over the drag button or the minimize/expand buttons

### CODE REVIEW

File: mockup/modalContents.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+<div class="modal">
+  <div class="modal-body">
+    <div class="modal-header">
+      <div class="tooltip center"><span class="material-icons modalIcon rotate">drag_indicator</span>
+        <span class="tooltiptext">Move window</span>
+      </div>
+      <span class="material-icons modalIcon right" for="o">close</span>
+      <div class="tooltip"><span class="material-icons modalIcon right">minimize</span>
+        <span class="tooltiptext">Minimize window</span>
+      </div>
```
Comment (SferaDev) [inline mockup/modalContents.html]: Move tooltip to react code

File: src/webapp/components/card-board/CardProgress.tsx [hunk 2]
```diff
@@ -18,6 +17,14 @@ export const CardProgress = styled.progress`
         border-radius: 15px;
     }
 
+    ::-moz-progress-bar {
+        	height:12px;
+        	width:100%;
+        	margin:0 auto;
+        	background-color: #43CBCB;
+        	border-radius: 15px;
+        }
```
Comment (SferaDev) [inline src/webapp/components/card-board/CardProgress.tsx]: Review

File: src/webapp/components/modal/ModalContent.tsx [hunk 3]
```diff
@@ -8,19 +8,22 @@ export const ModalContent = styled.div<{ bigger?: boolean }>`
 
     margin: 0;
     overflow-x: hidden;
-    overflow-y: auto;
+    overflow-y: scroll;
+    overflow-y: overlay;
+    scrollbar-width: thin;
+    scrollbar-color: #6894b5 transparent;
```
Comment (SferaDev) [inline src/webapp/components/modal/ModalContent.tsx]: Review color

File: src/webapp/components/modal/ModalContent.tsx [hunk 4]
```diff
@@ -8,19 +8,22 @@ export const ModalContent = styled.div<{ bigger?: boolean }>`
...
+        background: #276696;
+        border-radius: 6px;
     }
 
     ::-webkit-scrollbar-thumb {
-        background: #fff;
-        border-radius: 4px;
+        background: #6894b5;
+        border-radius: 6px;
     }
```
Comment (SferaDev) [inline src/webapp/components/modal/ModalContent.tsx]: Review color

**Review by ifoche — APPROVED** [review]
thanks @SferaDev

**Pull request merged**